### PR TITLE
follow c naming conventions

### PR DESCRIPTION
**Pull request by ZhijunZhao**
@zikalino @yaohaizh @xscript @aovechki

### CODE REVIEW

File: Lesson3/app/main.c [hunk 1]
```diff
@@ -21,59 +21,59 @@
...
-static int lastMessageSentTime = 0;
-static bool messagePending = false;
-static mraa_gpio_context context;
+static int g_total_blink_times = 1;
+static int g_last_message_sent_time = 0;
+static bool g_is_message_pending = false;
+static mraa_gpio_context g_context;
 
-int getTimeInSecond()
+int get_time_in_second()
```
Comment (zikalino) [inline Lesson3/app/main.c]: should be get_time_in_seconds

File: .travis.yml [hunk 2]
```diff
@@ -1,27 +1,14 @@
...
-        - yaweiw@microsoft.com
-        - yuwzho@microsoft.com
-        - reyang@microsoft.com
-        - zikalino@microsoft.com
-        - donzh@microsoft.com
-  
-matrix:
-  fast_finish: true
\ No newline at end of file
+language: cpp
```
Comment (xscript) [inline .travis.yml]: `c`?

File: .travis.yml [hunk 3]
```diff
@@ -1,27 +1,14 @@
...
+compiler: gcc
+
+install:
+  - sudo pip install cpplint
+
+script: 
+  - cpplint --filter=-whitespace/braces,-build/include_subdir,-whitespace/newline,-whitespace/line_length ./Lesson1/app/main.c ./Lesson3/app/main.c ./Lesson4/app/main.c  
+
```
Comment (xscript) [inline .travis.yml]: use wildcard to include all C source/header files?

File: Lesson3/app/main.c [hunk 4]
```diff
@@ -21,59 +21,59 @@
...
-    messagePending = false;
+    g_is_message_pending = false;
 }
 
-static void sendMessageAndBlink(IOTHUB_CLIENT_LL_HANDLE iotHubClientHandle, char *deviceId)
+static void send_message_and_blink(IOTHUB_CLIENT_LL_HANDLE iot_hub_client_handle, char *device_id)
 {
     char buffer[256];
-    sprintf(buffer, "{\"deviceId\":\"%s\",\"messageId\":%d}", deviceId, totalBlinkTimes);
+    snprintf(buffer, sizeof(buffer), "{\"device_id\":\"%s\",\"message_id\":%d}", device_id, g_total_blink_times);
```
Comment (xscript) [inline Lesson3/app/main.c]: should still use camel case in JSON

**Pull request merged**